### PR TITLE
Use comma instead of period for delimiter

### PR DESCRIPTION
**Pull request by Element21**
This will close Issue #23

### CODE REVIEW

**Comment by Tkd-Alex** [comment]
There is a bunch of things that I don't really like in this pull request sorry.
I'll review everything.

File: TwitchChannelPointsMiner/utils.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from TwitchChannelPointsMiner.constants.browser import USER_AGENTS
 
 
-def millify(input, precision=2):
+def Millify(input, precision=2):
```
Comment (Tkd-Alex) [inline TwitchChannelPointsMiner/utils.py:11]: The entire code base use the first letter in uppercase only for **Class** and not for **methods**.
So please change name or think up something else. For me also `_millify` It's good, but please don't use capital letter

Comment (Tkd-Alex) [reply TwitchChannelPointsMiner/utils.py:11]: Or `millify_numbers`

File: example.py [hunk 2]
```diff
@@ -8,32 +8,32 @@
 
 twitch_miner = TwitchChannelPointsMiner(
     username="your-twitch-username",
-    make_predictions=True,              # If you want to Bet / Make prediction | The browser will never start
-    follow_raid=True,                   # Follow raid to obtain more points
-    watch_streak=True,                  # If a streamer go online change the priotiry of streamers array and catch the watch screak. Issue #11
-    drops_events=True,                  # If you want to auto claim game drops from Twitch inventory Issue #21
-    claim_drops_startup=False,          # If you want to auto claim all drops from Twitch inventory on startup
+    make_predictions=True,  # If you want to Bet / Make prediction | The browser will never start
```
Comment (Tkd-Alex) [inline example.py:11]: About this I need that the example.py contains the same indention for readme (just for make it clear to read).
So, please copy and paste the current example.py from master and reset the indentation.

File: TwitchChannelPointsMiner/utils.py [hunk 3]
```diff
@@ -3,9 +3,13 @@
 
 from datetime import datetime, timezone
 from random import randrange
+import millify
```
Comment (Tkd-Alex) [inline TwitchChannelPointsMiner/utils.py:6]: `from millify import millify`

**Review by Tkd-Alex — CHANGES_REQUESTED** [review]
I've also configured pre-commit. So please do the following this before start again

`pip install pre-commit`
`pre-commit install`

This will be automatically lint and format the code with black.

File: TwitchChannelPointsMiner/utils.py (hunk 1 above)
Comment (Element21) [reply TwitchChannelPointsMiner/utils.py:11]: > The entire code base use the first letter in uppercase only for **Class** and not for **methods**.
> So please change name or think up something else. For me also `_millify` It's good, but please don't use capital letter

Definitely did not forget that... Sorry!

**Comment by Element21** [comment]
Ok, I just pushed two commits that fixes everything that you have requested. Can you please clear the changes requested?

**Review by Element21 — COMMENTED** [review]
These are outdated, since they are already fixed

**Review by Tkd-Alex — APPROVED** [review]

**Pull request merged**